src/linux/osd.c

```diff
@@ -71,6 +72,16 @@ ssize_t ofi_get_hugepage_size(void)
 	return val * 1024;
 }
 
+void ofi_fork_handlers(void)
+{
+	int ret;
+
+	ret = pthread_atfork(NULL, NULL, ofi_memhooks_atfork_handler);
```

Comment:
In similar circumstances, we would define pthread_atfork for the various OS's and just have them as no-ops.

Reply:
Will do.